### DISCUSSION TITLE
Is it possible to configure kcl to generate ".kclvm" and other kcl-related directories on a specific path only instead of $PWD?

### DISCUSSION THREAD

**Discussion by ealap** [kcl-lang/kcl#1322] [Q&A]
I created a small utility script that uses `kcl` to generate some configuration file for me. But wherever I invoke this script it leaves behind these directories.

If I understand correctly these are for caching but can we at least have them generated on a single pre-defined path aside from current directory?

**Comment by Peefy** [comment]
Yes, we can change the path through the variable.
export KCL_CACHE_PATH=/tmp # or change the path you want
